lblod/frontend-form-builder
# SFB-207: sharing functionality

## ID
 [SFB-207](https://binnenland.atlassian.net/browse/SFB-207)

 ## Description

When a user shares a form preview we do not want them to have certain elements on their screen.

Hide:
- topbar
- breadcrum bar
- link icon next to the title

Show:
* branding right under the test button (logo + Semantische formulieren)

 ## Type of change


 ## How to test

1. Press "voorbeeld" on a form in the overview
2. Share the form by clicking the link button
3. Open the link in a new tab
4. Only the form and branding should be visible

 ## Links to other PR's

 - /

 ## Attachments
![image](https://github.com/lblod/frontend-form-builder/assets/36266973/49289bdb-3b32-467a-8cdc-d0ba236db24f)
